tests/foreman/ui/test_contentcredentials.py

```diff
@@ -27,14 +26,9 @@
 empty_message = "You currently don't have any Products associated with this Content Credential."
 
 
-@pytest.fixture(scope='module')
-def module_org():
-    return entities.Organization().create()
-
-
 @pytest.fixture(scope='module')
 def gpg_content():
-    return DataFile.VALID_GPG_KEY_FILE.read_bytes()
+    return DataFile.VALID_GPG_KEY_FILE.read_text()
```

Comment:
What was the reason for this change?

Reply:
In most of the tests that used this method-
`TypeError: Object of type bytes is not JSON serializable`